numberGame.js

```diff
@@ -0,0 +1,37 @@
+import compareFunction from './compareFunction.js';
+
+const userInput = document.getElementById('enter-guess');
+const submitButton = document.getElementById('button');
+const triesRemaining = document.getElementById('tries-remaining');
+const box = document.getElementById('box');
+const winDisplay = document.getElementById('winDisplay');
+const lostDisplay = document.getElementById('lostDisplay');
+
+let tries = 4;
+
+const correctNumber = Math.round(Math.random() * 20) ;
```

Comment:
Looks like you function can randomly get 0 which would make the game impossible to win.